data.sql

```diff
@@ -1,4 +1,17 @@
 INSERT INTO animals VALUES (1, 'Agumon' ,'2020-2-5', 0, true,10.23);
 INSERT INTO animals VALUES (2, 'Gabumon' ,'2018-11-15', 2, true,8);
 INSERT INTO animals VALUES (4, 'Pikachu' ,'2021-1-7', 1, false,15.04);
-INSERT INTO animals VALUES (5, 'Devimon' ,'2017-5-12', 5, true,11);
\ No newline at end of file
+INSERT INTO animals VALUES (5, 'Devimon' ,'2017-5-12', 5, true,11);
+
+-- new data
+INSERT INTO animals VALUES (DEFAULT, 'Charmander', '2020-02-08', 0, '0', -11);
+INSERT INTO animals VALUES (DEFAULT, 'Plantmon', '2021-11-15', 3, '0', -5.7);
+INSERT INTO animals VALUES (DEFAULT, 'Squirtle', '1993-04-02', 1, '1', -12.13);
+INSERT INTO animals VALUES (DEFAULT, 'Angemon', '2005-06-12', 1, '1', -45.13);
+INSERT INTO animals VALUES (DEFAULT, 'Boarmon', '2005-06-07', 7, '1', 20.4);
+INSERT INTO animals VALUES (DEFAULT, 'Blossom', '1998-10-13', 3, '1', 17);
+INSERT INTO animals VALUES (DEFAULT, 'Ditto', '2022-05-14', 4, '1', 22);
+
+-- Add species to the table
+ALTER TABLE animals
+ADD species varchar(20)
```

Comment:
- [ ] Everything that creates or change the tables structure should be saved in the schema.sql file. this means that these lines should be in the schema.sql file.